.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -197,7 +207,7 @@ jobs:
         sudo bash -c 'echo 1 > /sys/kernel/debug/dri/0/i915_wedged'
     - uses: actions/checkout@v4
       with:
-        ref: ${{ inputs.ref }}
+        ref: ${{ inputs.devops_ref || inputs.ref }}
```

Comment:
another option - may be we can drop `ref ` at all so actions/checkout@v4 will use github.sha as by default.

Reply:
I think we still [want to] have CUDA on AWS testing. Due to its setup it can't use `pull_request` trigger and uses `pull_request_target` trigger. IIUC, the `ref` input is mostly to support that scenario (although I might be wrong here).

Reply:
I guess you mean this [one](https://github.com/intel/llvm/blob/sycl/.github/workflows/sycl-linux-precommit-aws.yml#L3)? As I understand it's not used for ~6months and we don't have AWS runners anymore or? Nick made these changes, so @sarnex can we just drop ref args at all? Like
```suggestion
```

Reply:
We still run it in Nightly: https://github.com/intel/llvm/blob/f3d12f0167daed873e25dc83309ed00acd3e591e/.github/workflows/sycl-nightly.yml#L147-L169

because we don't want the infrastructure support to rot.